### PR TITLE
Angle clarification proposal

### PR DESCRIPTION
**Pull request by cyrilleponcelet**
Hi, 
Please fin some clarification about angles, for discussion.
Note To my knowledge I do not kwown any sounder where the psi angles would be different from zero (or they should be constants maybe). Do you think it is better to remove the psi angles from variables definition ? they should be mentionned clearly in the matrix coordinates transforms in order for people to retrieve the euler angles matrix but we can suppose they are always equals to zero and clearly mention it in the corresponding chapter.

### CODE REVIEW

File: docs/tableBeamGroup1.adoc [hunk 1]
```diff
@@ -76,36 +76,36 @@ e|Variables | |
...
+ |{var}float rx_beam_rotation_theta(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise rotation about the _y_-axis of the platform coordinate system needed to give the receive beam coordinate system. For ships and similar, if installation angles are close to zero, this rotation usually matches the beam pointing angle in the along track direction (also called tilt angle).
  3+|{attr}:long_name = "receive beam angular rotation about the _y_ axis"
  3+|{attr}:units = "arc_degree" 
- 3+|{attr}float :valid_range = −90.0, 90.0
+ 3+|{attr}float :valid_range = −90.0, 90.0 
  
- |{var}float rx_beam_rotation_psi(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the receive beam coordinate system.
- 3+|{attr}:long_name = "receive beam angular rotation about the _z_ axis"
+ |{var}float rx_beam_rotation_psi(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the receive beam coordinate system. For most cases this angle is set to zero
```
Comment (gavinmacaulay) [inline docs/tableBeamGroup1.adoc:89]:
```suggestion
 |{var}float rx_beam_rotation_psi(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the receive beam coordinate system. For most cases this angle is set to zero.
```

File: docs/tableBeamGroup1.adoc [hunk 2]
```diff
@@ -76,36 +76,36 @@ e|Variables | |
...
+ 3+|{attr}:units = "arc_degree" 
  3+|{attr}float :valid_range = −90.0, 90.0
-
- |{var}float tx_beam_rotation_psi(ping_time, tx_beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the transmit beam coordinate system.
- 3+|{attr}:long_name = "transmit beam angular rotation about the _z_ axis"
- 3+|{attr}:units = "arc_degree"
- 3+|{attr}float :valid_range = −180.0, 180.0
-
+ 
+ |{var}float tx_beam_rotation_psi(ping_time, tx_beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the transmit beam coordinate system. For most cases this angle is set to zero
```
Comment (gavinmacaulay) [inline docs/tableBeamGroup1.adoc:104]:
```suggestion
 |{var}float tx_beam_rotation_psi(ping_time, tx_beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the transmit beam coordinate system. For most cases this angle is set to zero.
```

**Comment by gavinmacaulay** [comment]
The psi rotation helps to define the directions of split-aperture angles. These can change per beam. For example the convention with horizontal beams in Simrad omnionars is to have 'fore/aft' in the up/down direction. In our coordinate systems this is:
1. for a horizontal beam pointing forward, the 'fore/aft' coordinate is in the platform's x-z plane
2. for a horizontal beam pointing to starboard, the 'fore/aft' coordinate is in the platform's y-z plane

To achieve this, psi needs to be 0º in case 1 and 90º in case 2.

**Comment by cyrilleponcelet** [comment]
Clear explanation concerning the split aperture angle, I didn't knew that, Thanks. One of the big advantages to work in community

**Pull request merged**